stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -36,34 +36,6 @@ public protocol AnyActor: AnyObject {}
 /// point.
 @available(SwiftStdlib 5.5, *)
 public protocol DistributedActor: AnyActor, Identifiable, Hashable, Codable {
-
-    /// Creates new (local) distributed actor instance, bound to the passed transport.
-    ///
-    /// Upon initialization, the `id` field is populated by the transport,
-    /// with an address assigned to this actor.
-    ///
-    /// - Parameter transport: the transport this distributed actor instance will
-    ///   associated with.
-    init(transport: ActorTransport)
-
-    @available(*, deprecated, renamed: "SomeDistributedActor.resolve(_:using:)")
-    init(resolve id: AnyActorIdentity, using transport: ActorTransport) throws
-
```

Comment:
@kavon okey so the reason the conformance issue you saw yesterday is this:

- since we removed the inits from the protocol
- we won't run TypeChecker::addImplicitConstructors
- or at least we won't get to invoke inside it it seems `addImplicitDistributedActorMembersToClass`
- this entry point used to be for the inits, but as we now it moved on and actually synthesizes more and not just the inits

So removing the req, causes the synthesis to not trigger.

You can for now keep the silly init(transport: ActorTransport) in this PR and you should be unblocked.
I'm in the meantime working on fixing that in my local initializer rework.